SPECS/openblas/openblas.spec

```diff
@@ -1,7 +1,31 @@
-%bcond_with system_lapack
 # Version of bundled lapack
 %global lapackver 3.9.1
-
+# Do we have execstack?
+%global execstack 1
+# Build 64-bit interface binaries?
+%if 0%{?__isa_bits} == 64
+%global build64 1
+%bcond_without cpp_thread_check
+%else
+%global build64 0
+%bcond_with cpp_thread_check
+%endif
+%global base_description \
+OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD \
+version. The project is supported by the Lab of Parallel Software and \
+Computational Science, ISCAS. http://www.rdcps.ac.cn
+%bcond_with system_lapack
+# LAPACK
+%if %{with system_lapack}
+# Do we have LAPACKE? (Needs at least lapack 3.4.0)
+%global lapacke 1
+BuildRequires:  lapack-static
+%else
+# Use bundled LAPACK
+%global lapacke 1
+Provides:       bundled(lapack) = %{lapackver}
+%endif
```

Comment:
Please move this conditional `BuildRequires` and `Provides` below, next to the other ones.